swap/src/monero.rs

```diff
@@ -228,6 +240,21 @@ pub trait CreateWallet {
     async fn create_wallet(&self, file_name: &str) -> Result<()>;
 }
 
+#[async_trait]
+pub trait WalletBlockHeight {
```

Comment:
Are we ever using these traits meaningfully to create new abstractions? Differently asked: Is there a trait-bound somewhere that uses this trait?

If not, then I would propose to simplify this and just make these `pub` functions on the wallet type itself.

That doesn't break encapsulation aslong as all fields are still private, which they should be anyway, regardless of the traits.

Reply:
https://github.com/comit-network/xmr-btc-swap/pull/225/commits/0945cee459aeae17f3e7234ec3aa49cb26293c5d